src/lib/pubkey/ec_group/ec_point.h

```diff
@@ -223,6 +223,7 @@ class BOTAN_PUBLIC_API(2, 0) EC_Point final {
 #if defined(BOTAN_DISABLE_DEPRECATED_FEATURES)
 
    private:
+      friend class EC_Mul2Table_Data_BN;
```

Comment:
?

Reply:
> As a quick and easy (but maybe dirty) fix, I added this class as a friend in the deprecated case.

Reply:
Ah right, I think this got introduced in #4211 and we missed it due to not having #4051